src/Packlist/PacklistForm.js

```diff
@@ -98,24 +99,18 @@ export default function PacklistForm({ onPacklistSave }) {
           {errors.shouldHaveOneItem?.type === 'validate' && (
             <ErrorMessage>You need to add one item/task!</ErrorMessage>
           )}
-          {(errors.item?.type === 'validate' ||
-            errors.item?.type === 'minLength') && (
-            <ErrorMessage>
-              This field requires at least 3 characters!
-            </ErrorMessage>
-          )}
-          {(errors.item?.type === 'validate' ||
-            errors.item?.type === 'maxLength') && (
+
+          {itemError && (
             <ErrorMessage>
-              The Item can reach a maximum of 20 characters!
+              'This item field requires at least 3 characters and can reach a
+              maximum of 20 characters!'
             </ErrorMessage>
           )}
```

Comment:
Out of curiosity: why did you move the validation from react-hook-form to your own state?

Reply:
because react hook is not working here